app/src/main/java/com/dongyang/android/youdongknowme/ui/view/settingAlarm/SettingAlarmActivity.kt

```diff
@@ -0,0 +1,64 @@
+package com.dongyang.android.youdongknowme.ui.view.settingAlarm
+
+import android.content.Intent
+import android.os.Build
+import android.text.Spannable
+import android.text.SpannableStringBuilder
+import android.text.style.ForegroundColorSpan
+import android.widget.TextView
+import androidx.annotation.RequiresApi
+import com.dongyang.android.youdongknowme.R
+import com.dongyang.android.youdongknowme.databinding.ActivitySettingAlarmBinding
+import com.dongyang.android.youdongknowme.standard.base.BaseActivity
+import com.dongyang.android.youdongknowme.ui.view.main.MainActivity
+import com.dongyang.android.youdongknowme.ui.view.setting.SettingViewModel
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class SettingAlarmActivity : BaseActivity<ActivitySettingAlarmBinding, SettingViewModel>() {
+
+    override val layoutResourceId: Int = R.layout.activity_setting_alarm
+    override val viewModel: SettingViewModel by viewModel()
+
+    override fun initStartView() {
+        setSpan(binding.tvAlarmTitleMain,startIdx = 0, endIdx = 9)
+    }
+
+    override fun initDataBinding() {}
```

Comment:
사용하지 않는다면 오버라이딩 하지 않으셔도 됩니다 :)

Reply:
오버라이딩 필수...라서 안 넣으면 빌드 에러나요 큼.....

Reply:
앗 맞네요 ㅎㅎ..

```suggestion
    override fun initDataBinding() = Unit
```

으로 변경해주시면 될 것 같습니다!